api/src/main/java/org/pentaho/platform/api/genericfile/IGenericFileProvider.java

```diff
@@ -133,4 +139,13 @@ IGenericFileTree getTree( @NonNull GetTreeOptions options )
    * @return {@code true}, if the provider owns the specified generic file path; {@code false}, otherwise.
    */
   boolean owns( @NonNull GenericFilePath path );
+
+  /**
+   * Checks whether a generic file exists and the current user has the specified permissions on it.
+   * ...
```

Comment:
Missing doclet parameters (instead of the `...`).

Reply:
fixed